deploy/examples/create-external-cluster-resources.py

```diff
@@ -474,6 +474,24 @@ def gen_arg_parser(cls, args_to_parse=None):
             required=False,
             help="provides the name of the rgw-zonegroup",
         )
+        output_group.add_argument(
```

Comment:
support for rados-namspace for replica-n pools would be considered in follow PR if requested as discussed